lib/zstream/protocol.ex

```diff
@@ -186,7 +182,7 @@ defmodule Zstream.Protocol do
     # encryption bit set based on coder
     # bit 3 use data descriptor
     # bit 11 UTF-8 encoding of filename & comment fields
-    <<encryption_coder.general_purpose_flag() ||| 0x0008 ||| 0x0800::little-size(16)>>
+    encryption_coder.general_purpose_flag() ||| 0x0008 ||| 0x0800
```

Comment:
any reason for removing `::little-size(16)`?

Reply:
Yes, it's included in the binary where this function is called.